cmd/kubeadm/app/cmd/upgrade/common.go

```diff
@@ -131,23 +128,12 @@ func enforceRequirements(flags *applyPlanFlags, args []string, dryRun bool, upgr
 	}
 
 	// Fetch the configuration from a file or ConfigMap and validate it
-	printer.Printf("[upgrade/config] Making sure the configuration is correct:\n")
+	_, _ = printer.Println("[upgrade/config] Loading the kubeadm configuration")
```

Comment:
the combination of the linter errors for missing return values and our printer API which strangely returns ints and errors, is a bit annoying.

Reply:
+1, I must do something similar here: https://github.com/kubernetes/kubernetes/pull/114062/files#diff-657870bffd956e3e3553db7dbb5cb725ea53922da959fcf869c2e31880df72c7R66 :)